Shared/View/VoteView.swift

```diff
@@ -60,6 +82,7 @@ struct VoteView: View {
 
 struct VoteView_Previews: PreviewProvider {
     static var previews: some View {
-        VoteView()
+        // TODO: Need to create ActicityID and OptionID, store in variable, and pass it in here
+        VoteView().environmentObject(UserViewModel()).environmentObject(ActivityViewModel(activityID: "AhdjSLOlTJqb68aXBAWn")).environmentObject(OptionViewModel(optionID: "B5uYuFf31aEqH5sGHFA4"))
```

Comment:
Yeah, most likely you'd want VoteView to take the activity ID as a parameter and construct ActivityViewModel instance based on that. You may need to write a custom initializer (i.e. constructor)